pytest/testsuites/test_lis.py

```diff
@@ -0,0 +1,19 @@
+"""Runs 'LIS-Tests.xml' using Pytest."""
+import conftest
+import pytest
+from node_plugin import Node
+
+
+@pytest.mark.lisa(
```

Comment:
Can it be written like "lisa_case" without "pytest.mark"? Test case is our product, a simpler name is more meaningful.

Reply:
Yes, of course...this is just Python...it would look like:

```python
lisa = pytest.mark.lisa # put this anywhere you want, conftest for now, lisa plugin when we do that


@lisa(
    platform="Azure", category="Functional", area="LIS_DEPLOY", tags=["lis"], priority=0
)
```